### PR TITLE
[DO NOT MERGE] DEVELOPER-3853 - Create a script to push existing videos into Drupal

### PR DESCRIPTION
**Pull request by Dantheman720**
Testing Phinx's database migration tool. This should remove all video nodes. We will test pushing the videos to this pull request.

### CODE REVIEW

**Comment by LightGuard** [comment]
How did this work for you, Dan?

**Comment by Dantheman720** [comment]
retest this please

**Comment by Dantheman720** [comment]
retest this please.

**Comment by Dantheman720** [comment]
retest this please

File: _docker/environments/drupal-dev/rhd.settings.php [hunk 1]
```diff
@@ -4,8 +4,8 @@
 $settings['container_yamls'][] = DRUPAL_ROOT . '/sites/development.services.yml';
 
 $settings['twig_debug'] = TRUE;
-$config['system.performance']['css']['preprocess'] = FALSE;
-$config['system.performance']['js']['preprocess'] = FALSE;
+$config['system.performance']['css']['preprocess'] = TRUE;
```
Comment (LightGuard) [inline _docker/environments/drupal-dev/rhd.settings.php]: All of these changes need to go away

File: _docker/environments/drupal-dev/rhd.settings.php [hunk 2]
```diff
@@ -34,4 +34,4 @@
 $settings['install_profile'] = 'lightning';
 
 /* Increase default memory settings for Drupal to 256 meg. Taken from: https://www.drupal.org/docs/7/managing-site-performance-and-scalability/changing-php-memory-limits */
-ini_set('memory_limit', '256M');
\ No newline at end of file
+ini_set('memory_limit', '2560M');
```
Comment (LightGuard) [inline _docker/environments/drupal-dev/rhd.settings.php]: And here

File: _ext/jboss_developer.rb [hunk 3]
```diff
@@ -578,6 +579,7 @@ def post(endpoint, path, params = {})
             @logger.debug "request body: #{req.body}"
           end
         end
+        resp
```
Comment (LightGuard) [inline _ext/jboss_developer.rb]: Really doesn't seem necessary. Ruby returns whatever is at the end of a block.

File: _ext/temp.json [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+      "lang": "en"
+    }
+  ],
+  "status": [
+    {
+      "value": "1",
+      "lang": "en"
+    }
+  ]
+}
```
Comment (LightGuard) [inline _ext/temp.json]: Is this needed?

File: _config/site.yml [hunk 5]
```diff
@@ -89,8 +91,9 @@ profiles:
     dcp_base_protocol_relative_url: https://dcp.stage.jboss.org/
     base_url: http://docker/
     dcp_base_protocol_relative_url: https://docker:<%= ENV['SEARCHISKO_HOST_PORT'] %>/
-    drupal_base_url: http://drupal/
+    drupal_base_url: http://docker:8888/
```
Comment (LightGuard) [inline _config/site.yml]: Maybe not the correct change here.

File: _docker/drupal/drupal-filesystem/scripts/run-httpd.sh [hunk 6]
```diff
@@ -18,4 +18,5 @@ rm -rf /run/httpd/*
 chown -R apache:apache /var/www/drupal/web/sites
 chown -R apache:apache /var/www/drupal/web/config/active # just to make sure we can write changes to active
 
+drush cr all
```
Comment (LightGuard) [inline _docker/drupal/drupal-filesystem/scripts/run-httpd.sh]: You may need to add  "--root=/var/www/drupal/web" here.

File: _docker/environments/drupal-dev/docker-compose.yml [hunk 7]
```diff
@@ -4,8 +4,7 @@ services:
     build: ../../awestruct
     command:
       - "rake git_setup clean gen[drupal_dev]"
-    links:
-      - drupal
+    network_mode: "host"
```
Comment (LightGuard) [inline _docker/environments/drupal-dev/docker-compose.yml]: This was a change for your environment, shouldn't have been committed.

File: _ext/pipeline.rb [hunk 8]
```diff
@@ -165,20 +168,20 @@
...
-                                                       worksheet_title: 'YouTube Videos',
-                                                       col_labels: true,
-                                                       by: 'row')
-  extension Aweplug::Extensions::Video.new("site.youtube.collect {|i,v| v['youtube_url']}", site.push_to_searchisko)
+  # extension Aweplug::Extensions::GoogleSpreadsheet.new(assign_to: 'youtube',
+  #                                                      key: '1QbjVeU9avP8hcnaruiLtuanQVpkdClIYtgFSmaC_K9c',
+  #                                                      worksheet_title: 'YouTube Videos',
+  #                                                      col_labels: true,
+  #                                                      by: 'row')
+  # extension Aweplug::Extensions::Video.new("site.youtube.collect {|i,v| v['youtube_url']}", site.push_to_searchisko)
```
Comment (LightGuard) [inline _ext/pipeline.rb]: I'm not sure why we're commenting out all the video spreadsheets.

**Pull request closed without merging**